packages/wallet/src/wallet.test.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
is this the same fixtures as `fuels-rs` is using too? might be good to sync them

Reply:
Yes, I got it from `fuels-rs`. I was thinking about creating some way to share via module. Maybe create a repo that is a module for `rust` and `typescript`.